examples/release_highlights/plot_release_highlights_1_2_0.py

```diff
@@ -27,7 +27,7 @@
 # scikit-learn's transformers now support pandas output with the `set_output` API.
 # To learn more about the `set_output` API see the example:
 # :ref:`sphx_glr_auto_examples_miscellaneous_plot_set_output.py` and
-# this `YouTube video <https://www.youtube.com/watch?v=J4KCu9WWDTo>`__.
+# this `YouTube video <https://youtu.be/5bCg8VfX2x8>`__.
```

Comment:
@thomasjpfan Seems like I don't have rights to suggest edits to this PR.

In any case, can we change:
```
# this `YouTube video <https://youtu.be/5bCg8VfX2x8>`__.
```
to 
```
# this `video, pandas DataFrame output for scikit-learn transformers (some examples) <https://youtu.be/5bCg8VfX2x8>`__.
```

See note on accessible hyperlinks:  https://github.com/data-umbrella/dasch/issues/6#issue-1318180916